hopp/simulation/technologies/steel/enthalpy_functions.py

```diff
@@ -0,0 +1,194 @@
+## Functions to calculate enthalpy of different streams 
+
+### Hydrogen enthalpy coefficients
+# The value of enthalpy returned is the specific enthalpy in kj/g
+'''
+Author: Charlie Kiefer
+Date: 8/14/23
+
+These functions calculate enthalpies of elements relating to green steel
+
+Sources:
+Chase, M.W., Jr. 1998. "NIST-JANAF Themochemical Tables, Fourth Edition." J. Phys. Chem. Ref. Data, Monograph 9 1-1951. doi:https://doi.org/10.18434/T4D303.
+    Thermochemistry tab of each element
+
+Inputs:
+    T: Temperature of the element (K)
+
+Outputs:
+    H_t: Enthalpy of element at given temp (kj/g)
+'''
+
+def h2_enthalpy_1(T):
+    mol_weight_H2=2.01588 #in grams# T1 and T2 should be in the range of 298-1000 K <br>
+    t=T/1000 
+    A=33.066718 
+    B=-11.363417
+    C=11.432816 
+    D=-2.772874 
+    E=-0.158558 
+    F=-9.980797 
+    G=172.707974 
+    H=0  
+    H_t=A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H 
+    return H_t 
+def h2_enthalpy_2(T):
```

Comment:
The h2_enthalpy_x functions should be a single function with an if statement to determine which method to use. You could to this by writing a third function called `h2_enthalpy` that then calls the existing two functions, or you can just combine the contents of the two functions.

Reply:
Merged into one function with if statements